Promote Handles to RATS Conceptual Messages (#90)
ietf-rats-wg/draft-ietf-rats-reference-interaction-models: draft-ietf-rats-reference-interaction-models.md

If there is no way to convey a Claim Selection in a remote attestation protocol, a default Claim Selection (e.g., "all") MUST be defined by the Attester and SHOULD be known by the Verifier.
In order to select Claims, Claims that can be potentially included in Evidence by an Attesting Environment have to be known by the Verifier.

## Handles as RATS Conceptual Messages {#cm-handle}

Handles, whether instantiated as attestation nonces or Epoch IDs, are fundamental to many Remote Attestation protocols.
In all the interaction models presented in this document, Handles are exchanged between RATS roles and used to correlate and lock together the different stages of the interaction.
Given their ubiquity and key role, it is appropriate to recognize them as RATS Conceptual Message in their own right.

Accordingly, {{iana-handle-cmind}} registers a new Conceptual Message indicator for Handles to be used with Conceptual Message Wrappers (CMW) {{-rats-cmw}}.

> Note that concrete Handle representations require an associated media type for use with CMW.

Reviewer: ```suggestion
A concrete Handle representations requires an associated media type for use with CMW.
```

By addressing these application-specific security requirements within the context of defined interaction models, security strategies can be tailored to fit the unique challenges and operational contexts of different attestation scenarios.

# IANA Considerations

## New Conceptual Message Indicator for Handles {#iana-handle-cmind}

> IANA is requested to add a new Indicator Value to the "RATS Conceptual Message Wrapper (CMW) Indicators" registry of the "Remote Attestation Procedures (RATS)" registry group {{IANA.rats}} as described in {{tab-handle-cmind}}.

Author: ```suggestion
IANA is requested to add a new entry to the "RATS Conceptual Message Wrapper (CMW) Indicators" registry of the "Remote Attestation Procedures (RATS)" registry group {{IANA.rats}} as described in {{tab-handle-cmind}}.
```
